include/LightGBM/R_object_helper.h

```diff
@@ -17,48 +19,47 @@
 // use .Internal(internalsID()) to uuid
 #define R_INTERNALS_UUID "2fdf6c18-697a-4ba7-b8ef-11c0d92f1327"
 
-
 #ifdef R_VER_ABOVE_35
-#define NAMED_BITS 16
-struct lgbm_sxpinfo {
-  unsigned int type : 5;
-  unsigned int scalar : 1;
-  unsigned int obj : 1;
-  unsigned int alt : 1;
-  unsigned int gp : 16;
-  unsigned int mark : 1;
-  unsigned int debug : 1;
-  unsigned int trace : 1;
-  unsigned int spare : 1;
-  unsigned int gcgen : 1;
-  unsigned int gccls : 3;
-  unsigned int named : NAMED_BITS;
-  unsigned int extra : 32 - NAMED_BITS;
-};
-
-// 64bit pointer
-#if INTPTR_MAX == INT64_MAX
-typedef int64_t R_xlen_t;
-#else
-typedef int R_xlen_t;
-#endif
+  #define NAMED_BITS 16
+  struct lgbm_sxpinfo {
+    unsigned int type : 5;
+    unsigned int scalar : 1;
+    unsigned int obj : 1;
+    unsigned int alt : 1;
+    unsigned int gp : 16;
+    unsigned int mark : 1;
+    unsigned int debug : 1;
+    unsigned int trace : 1;
+    unsigned int spare : 1;
+    unsigned int gcgen : 1;
+    unsigned int gccls : 3;
+    unsigned int named : NAMED_BITS;
+    unsigned int extra : 32 - NAMED_BITS;
+  };
+
+  // 64bit pointer
+  #if INTPTR_MAX == INT64_MAX
+    typedef int64_t R_xlen_t;
+  #else
+    typedef int R_xlen_t;
+  #endif
 
 #else
-struct lgbm_sxpinfo {
-  unsigned int type : 5;
-  unsigned int obj : 1;
-  unsigned int named : 2;
-  unsigned int gp : 16;
-  unsigned int mark : 1;
-  unsigned int debug : 1;
-  unsigned int trace : 1;
-  unsigned int spare : 1;
-  unsigned int gcgen : 1;
-  unsigned int gccls : 3;
-};
-
-typedef int R_xlen_t;
-#endif
+  struct lgbm_sxpinfo {
+    unsigned int type : 5;
+    unsigned int obj : 1;
+    unsigned int named : 2;
+    unsigned int gp : 16;
+    unsigned int mark : 1;
+    unsigned int debug : 1;
+    unsigned int trace : 1;
+    unsigned int spare : 1;
+    unsigned int gcgen : 1;
+    unsigned int gccls : 3;
+  };
+
+  typedef int R_xlen_t;
+#endif // R_VER_ABOVE_35
```

Comment:
Please fix this:
```
include/LightGBM/R_object_helper.h:62:  At least two spaces is best between code and comments  [whitespace/comments] [2]
include/LightGBM/R_object_helper.h:191:  At least two spaces is best between code and comments  [whitespace/comments] [2]
```

Reply:
oh sure! Cannot wait until we enable `cpplint` fully in CI

Reply:
Me too!